task-list/config/routes.rb

```diff
@@ -1,25 +1,37 @@
 Rails.application.routes.draw do
-  # home page
-  root 'tasks#index'
-  get 'tasks/' => 'tasks#index'
+# task routes:
+    # home page
+    root 'tasks#index'
+    get 'tasks/' => 'tasks#index'
 
-  # complete/uncomplete task
-  patch '/tasks/:id/complete/' => 'tasks#complete'
-  patch '/tasks/:id/incomplete/' => 'tasks#incomplete'
+    # complete/uncomplete task
+    patch '/tasks/:id/complete/' => 'tasks#complete'
+    patch '/tasks/:id/incomplete/' => 'tasks#incomplete'
 
-  # new task
-  get 'tasks/new' => 'tasks#new'
-  post 'tasks/' => 'tasks#create'
+    # new task
+    get 'tasks/new' => 'tasks#new'
+    post 'tasks/' => 'tasks#create'
 
-  # show task
-  get 'tasks/:id' => 'tasks#show', as: :task
+    # show task
+    get 'tasks/:id' => 'tasks#show', as: :task
 
-  # delete task
-  delete 'tasks/:id' => 'tasks#destroy'
+    # delete task
+    delete 'tasks/:id' => 'tasks#destroy'
+
+    # edit task
+    get 'tasks/:id/edit' => 'tasks#edit'
+    patch 'tasks/:id/' => 'tasks#update'
```

Comment:
Should this route's URL path end with a `/`? None of the other routes that use `:id` end with one.